docs/versioned_docs/version-0.2.2/api/06-Configuration/askui-ui-control-client.md

```diff
@@ -64,14 +64,14 @@ Setting credentials in code:
 Setting credentials using environment variables (Powershell):
 ```powershell
 $env:ASKUI_TENANT = <tenat>
-$env:ASKUI_TOKEN = <token>
+$env:ASKUI_TOKEN = <access token>
```

Comment:
Just an idea: I think it would be nice to have something like a dictionary where we can add commonly used terms so that we are more consistent.

Reply:
And it would be nice if Docusaurus to display tooltips when those terms are used. Providing the user with an instant explanation instead of going to a Glossar of some sort.

I will add this to our backlog!

Reply:
Absolutely, love the idea :)